torchtext/models/t5/modules.py

```diff
@@ -30,9 +30,8 @@
 PAST_KEY_VALUES_UNFILLED_TYPE = Tuple[Tensor, Tensor, Optional[Tensor], Optional[Tensor]]
 
 # Define reusable types for encoder/decoder outputs
-ENCODER_OUTPUTS_TYPE = Dict[str, Union[Optional[Tensor], List[Tensor], List[Optional[Tensor]]]]
-DECODER_OUTPUTS_TYPE = Dict[
-    str, Union[Optional[Tensor], List[Tensor], List[Optional[Tensor]], List[PAST_KEY_VALUES_UNFILLED_TYPE]]
+SEQ_2_SEQ_OUTPUTS_TYPE = Dict[
+    str, Union[Optional[Tensor], List[Tensor], List[Optional[Tensor]], Optional[List[PAST_KEY_VALUES_UNFILLED_TYPE]]]
```

Comment:
Yes, this is not an ideal solution. I'm open to ideas on how to generalize here, but figured I'd just get it working rather than over-optimizing right now.